gbpservice/nfp/configurator/drivers/base/base_driver.py

```diff
@@ -28,25 +30,38 @@ class BaseDriver(object):
     def __init__(self, conf):
         pass
 
+    def configure_healthmonitor(self, context, resource_data):
+        """Checks if the Service VM is reachable.
+
+           It does netcat to the CONFIGURATION_SERVER_PORT of the Service VM.
+           Configuration agent runs inside Service VM. Once agent is up and
+           reachable, Service VM is assumed to be active.
+
+           :param context - context
+           :param resource_data - data coming from orchestrator
+
+           Returns: SUCCESS/FAILED
+
+        """
+        ip = resource_data.get('mgmt_ip')
+        port = str(self.port)
```

Comment:
Since no one calls this method from the driver, we can't pass it. Instead we use the port defined in the main driver class.